.travis.yml

```diff
@@ -13,6 +13,7 @@ cache:
 
 before_install:
   - mv ~/.phpenv/versions/$(phpenv version-name)/etc/conf.d/xdebug.ini{,.disabled} || echo "xdebug not available"
+  - composer validate
   - travis_retry composer self-update
   - travis_retry composer config --unset platform.php
```

Comment:
I think we need to remove this line now, right?

Reply:
oh yes